modules/nf-core/dragmap/align/environment.yml

```diff
@@ -0,0 +1,8 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::dragmap=1.2.1
+  - bioconda::samtools=1.15.1
```

Comment:
Even older samtools version, but might not be worth the update here. I think we are still waiting for a new dragmap release that doesn't result in a segmentation fault

Reply:
https://github.com/nf-core/modules/issues/4288